incubator/java-microprofile/image/Dockerfile-stack

```diff
@@ -21,9 +21,11 @@ RUN mvn -Dmaven.repo.local=/mvn/repository install dependency:go-offline -DskipT
 
 WORKDIR /project/user-app
 
-ENV APPSODY_MOUNTS="~/.m2/repository:/mvn/repository;src:/project/user-app/src;pom.xml:/project/user-app/pom.xml"
+ENV APPSODY_MOUNTS="~/.m2/repository:/mvn/repository;.:/project/user-app"
```

Comment:
We explored this in another PR and there's a performance concern.    Why did we need this?

Reply:
I realized this is probably to get around the non-root user having access to root-owned project structure (e.g. target/classes).  

Maybe we should do something like the SB2 stack:  https://github.com/appsody/stacks/blob/deb09f9d8a404b6105a4ee4e56ebce6d9c5e6614/incubator/java-spring-boot2/image/Dockerfile-stack#L10-L21